pkg/secrets/pushtofile/secret_group.go

```diff
@@ -301,3 +311,26 @@ func newSecretGroup(groupName string, secretsBasePath string, annotations map[st
 
 	return sg, nil
 }
+
+func validateGroupFilePaths(secretGroups []*SecretGroup) []error {
+	// Iterate over the secret groups and group any that have the same file path
+	groupFilePaths := make(map[string][]string)
+	for _, sg := range secretGroups {
+		if groupFilePaths[sg.FilePath] == nil {
+			groupFilePaths[sg.FilePath] = []string{sg.Name}
+		} else {
+			groupFilePaths[sg.FilePath] = append(groupFilePaths[sg.FilePath], sg.Name)
+		}
```

Comment:
To retain a [line of sight](https://github.com/cyberark/community/blob/main/Conjur/conventions/golang-style-guide.md#line-of-sight-coding-best-practice)
```
		if len(groupFilePaths[sg.FilePath]) == 0 {
			groupFilePaths[sg.FilePath] = []string{sg.Name}
                        continue
		} 

	        groupFilePaths[sg.FilePath] = append(groupFilePaths[sg.FilePath], sg.Name)
```